Prox/Prox/PlaceDetails/PlaceDetailsCardViewController.swift

```diff
@@ -101,9 +102,68 @@ class PlaceDetailsCardViewController: UIViewController {
             return
         }
         cardView.updateUI(forPlace: place)
+        setLocation(location: locationProvider?.getCurrentLocation())
+        setupCardInteractions()
+
         pageControl.numberOfPages = place.photoURLs?.count ?? 0
     }
 
+    private func setLocation(location: CLLocation?) {
+        if let lastTravelTimes = place.lastTravelTime {
```

Comment:
We should add a comment here (and in the similar code) that if we're not cached and a request is in motion, we're not waiting for a response – we're creating a new request.